rss2email/main.py

```diff
@@ -200,6 +200,7 @@ def run(*args, **kwargs):
         if not args.verbose:
             _LOG.setLevel(feeds.config['DEFAULT']['verbose'].upper())
         args.func(feeds=feeds, args=args)
+        feeds.close()
```

Comment:
Looking at [the wider context](https://github.com/auouymous/rss2email/blob/e6ff9db705b233f63692610cdbe490ab293d0721/rss2email/main.py#L194-L211):
```python
    try:
        if not args.config:
            args.config = None
        feeds = _feeds.Feeds(datafile_path=args.data, configfiles=args.config)
        if args.func != _command.new:
            feeds.load()
        if not args.verbose:
            _LOG.setLevel(feeds.config['DEFAULT']['verbose'].upper())
        args.func(feeds=feeds, args=args)
        feeds.close()
    except _error.RSS2EmailError as e:
        e.log()
        if _logging.ERROR - 10 * args.verbose < _logging.DEBUG:
            raise  # don't mask the traceback
        _sys.exit(1)
    finally:
        if lockfile is not None:
            lockfile.close()
```
We know now that `if args.func != _command.new:` is redundant (because `_command.new` is not the only exception), and `feeds.close()` could arguably move to the `finally:` clause.

Reply:
I agree that `feeds.close()` should be moved to finally. But are you suggesting the `_command.new` conditional is redundant because pause and unpause also don't require feeds to be loaded?

Reply:
I meant it was redundant because we are now closing the file anyway; of course it is better to not open the file in the first place if it is not needed. My [original suggestion](https://github.com/rss2email/rss2email/issues/204#issuecomment-1030767777) was to not call `feeds.load()` here at all, and instead call it only from the places that need it in `rss2email/command.py`. This would be better and more in line with the Zen of Python (“explicit is better than implicit”) than either removing this condition on `args.func != _command.new` or updating it to include all of the exceptions (currently `new`, `email`, `list`, `pause`, `unpause`, `opmlexport`).

Reply:
I agree, it would be better to only load the feeds inside each command that needs them. Do you want to submit a PR for that? Or I can try to get to it in a few days if not.